lib/rspotify/player.rb

```diff
@@ -71,6 +71,20 @@ def play(device_id = nil, params = {})
       User.oauth_put(@user.id, url, params.to_json)
     end
 
+    # Add an item to the end of the user’s current playback queue
+    # If `device_id` is not passed, the currently active spotify app will be triggered
+    #
+    # @example
+    #           player = user.player
+    #           player.queue(nil, "spotify:track:4iV5W9uYEdYUVa79Axb7Rh")
+    def queue(device_id = nil, uri)
+      url = "me/player/queue"
+      url = device_id.nil? ? url : "#{url}?device_id=#{device_id}"
+      param = {"uri": uri}
+
+      User.oauth_put(@user.id, url, param.to_json)
```

Comment:
This API endpoint is a `POST` not a `PUT`. [reference](https://developer.spotify.com/documentation/web-api/reference-beta/#endpoint-add-to-queue)
```suggestion
      User.oauth_post(@user.id, url, {})
```